pytest_splunk_addon/standard_lib/data_models/Network_Traffic.json

```diff
@@ -186,7 +197,7 @@
         },
         {
           "name": "rule",
-          "type": "optional",
+          "type": "required",
```

Comment:
this is also not realistic. missing 74 times https://github.com/search?q=org%3Asplunk%20%20%3Cfield%3Erule%3C%2Ffield%3E&type=code and populated 55 times org:splunk  <field name="rule" value="

Reply:
Please revert `rule` as optional. I made a mistake in the sheet (i just corrected it in the sheet)

Reply:
done